### PR TITLE
Fix update subscriptions dev

### PR DESCRIPTION
**Pull request by angrykoala**
This fix removes all references to meta in the updates query itself. If subscriptions are set, it will wrap the whole block in a call statement, with the pre and post assignments of the meta variables

This PR depends on #1583

### CODE REVIEW

**Comment by neo4j-team-graphql** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-graphql-docs-1584.surge.sh

File: packages/graphql/tests/tck/tck-test-files/subscriptions/update.test.ts [hunk 1]
```diff
@@ -73,7 +73,12 @@ describe("Subscriptions metadata on update", () => {
             MATCH (this:Movie)
             WHERE this.id = $this_id
             WITH this { .* } AS oldProps, this, meta
-            SET this.id = $this_update_id
+            CALL {
+            	WITH *
+            	SET this.id = $this_update_id
+            	RETURN meta as update_meta
+            }
+            WITH *, update_meta as meta
```
Comment (darrellwarde) [inline packages/graphql/tests/tck/tck-test-files/subscriptions/update.test.ts:81]: Is there any value in this? Could it not have just been:

```
SET
SET
SET
WITH this, meta + { ...meta }
```

I guess I'm asking, was this possible without the subquery?

**Review by darrellwarde — APPROVED** [review]
It looks good, I just have a question around the choice of using a subquery here, but it's an uneducated question!

File: packages/graphql/tests/tck/tck-test-files/subscriptions/update.test.ts (hunk 1 above)
Comment (angrykoala) [reply packages/graphql/tests/tck/tck-test-files/subscriptions/update.test.ts:81]: The problem without subquery is that oldProps will pollute the nested queries, the elegant thing about `CALL { WITH *` is that it completely encapsulates subqueries

Comment (darrellwarde) [reply packages/graphql/tests/tck/tck-test-files/subscriptions/update.test.ts:81]: I have no major objection, just a curiosity thing!

**Pull request merged**